sample-kotlin/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion COMPILE_SDK_VERSION
+    buildToolsVersion BUILD_TOOLS_VERSION
+    defaultConfig {
+        applicationId "permissions.dispatcher.samplekotlin"
```

Comment:
For consistency, I'd like the package name to be consistent with the Java sample: `permissions.dispatcher.sample`. The module already tells everything the user needs to know about this sample being written in Kotlin.